CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+## 1.27.0
+
+This minor release contains bugfixes and other improvements related to the tests and the documentation.
+
+## New Features
+- stage1/kvm: add arm64 build ([#3690](https://github.com/rkt/rkt/pull/3690)).
+
+## Bugfixes
+- stage0: list|status --format=json panics: RuntimeApp.Mounts.AppVolume is optional ([#3699](https://github.com/rkt/rkt/pull/3699)). When it is nil, the Volume info at the Pod level (with the same name) should be used. Without this patch `rkt list --format=json` panics on a nil pointer when Apps reference Volumes from the Pod level.
+- imagestore: Fix sql resource leaks ([#3682](https://github.com/rkt/rkt/pull/3682)). When using sql queries the rows iterator needs to be closed if the entire query  result is not iterated over. Failure to close the iterator results in resource leakage.
+
+## Other changes
+- networking: change the default-restricted subnet ([#3718](https://github.com/rkt/rkt/pull/3718)). Previously, we were using 172.17/16, which conflicts with the default
+Docker networking. Change it to 172.31/16.
+- scripts/pkg: improved detection of active mounts ([#3710](https://github.com/rkt/rkt/pull/3710)). On systems which have /var/lib/rkt as a separate partition, the active mount detection in before-remove needs to not get confused by the presence of /var/lib/rkt itself as a mount.  Therefore a longer path is used for active mount detection.
+- docs: fix code block syntax ([#3709](https://github.com/rkt/rkt/pull/3709)). This fixes some rendering issues on https://coreos.com/rkt/docs/latest/commands.html#logging.
+- Documentation: CoreOS -> Container Linux ([#3700](https://github.com/rkt/rkt/pull/3700)).
+- docs: how to get password for the ubuntu user ([#3679](https://github.com/rkt/rkt/pull/3679)). Mac OS 10.12.3 Vagrant 1.9.4 VirtualBox 5.0.8
```

Comment:
Normaly, we coalesce all documentation updates in to a single list. If you don't like to do that, could you put them in a separate category?